src/test/java/com/elasticbox/jenkins/tests/PullRequestTestBase.java

```diff
@@ -96,14 +103,40 @@
     public void setup() throws Exception {
         super.setup();
         boolean customApiUrl;
+ //       String backWebHookUrlLocalhost = null;
+        String backJenkinsUrl = null;
+
         if (TestUtils.GITHUB_ADDRESS.equals(MessageFormat.format("https://{0}", TestUtils.GITHUB_PUBLIC_ADDRESS))) {
             apiGithubAddress = MessageFormat.format("https://api.{0}", TestUtils.GITHUB_PUBLIC_ADDRESS);
             customApiUrl = false;
         } else {
             apiGithubAddress = MessageFormat.format("{0}/api/v3", TestUtils.GITHUB_ADDRESS);
             customApiUrl = true;
         }
-        webhookUrl = ((PullRequestBuildTrigger.DescriptorImpl) jenkinsRule.getInstance().getDescriptor(PullRequestBuildTrigger.class)).getWebHookUrl();
+
+        PullRequestBuildTrigger.DescriptorImpl descriptor = (PullRequestBuildTrigger.DescriptorImpl) jenkinsRule.getInstance().getDescriptor(PullRequestBuildTrigger.class);
```

Comment:
I don't like this fragment too much. I guess there should be a better way to handle it, without having to get the Descriptor itself... Not a blocker for this one, but let's try to refactor this for the next one.